grid/recruitment/list-candidates/maps/greenhouse.suma

```diff
@@ -0,0 +1,49 @@
+profile = "recruitment/list-candidates@1.0"
+provider = "greenhouse"
+
+map ListCandidates {
+  http GET "/v1/candidates" {
+    security "apiKeyBasicAuth"
+    
+
+    request {
+      query {
+        job_id = input.jobId
+        page = input.page
+        per_page = input.limit
+      }
+    }
+
+
+    response 200 "application/json" {
+      map result {
```

Comment:
There is no `nextPage` in the result object, meaning the user cannot go to the next page of the result.

Reply:
Nevermind, I completely missed it!